src/noit_lmdb_tools.c

```diff
@@ -284,6 +284,14 @@ noit_lmdb_instance_t *noit_lmdb_tools_open_instance(char *path)
     return NULL;
   }
 
+  /* Set initial pagesize to 64 MiB */
```

Comment:
This is the map size, not the page size, correct?

Reply:
Yup, typo. Fixed.